atmos.yaml

```diff
@@ -17,6 +17,12 @@
 # are considered paths relative to 'base_path'.
 base_path: "./examples/quick-start-advanced"
 
+vendor:
+  # Path to the vendor.yaml file
+  # Supports both absolute and relative paths
+  # Can also be set using 'ATMOS_VENDOR_YAML_PATH' ENV var, or '--vendor-yaml-path' command-line argument
+  vendor_yaml_path: "./vendor.yaml"
```

Comment:
@Cerebrovinny can you update this so it supports either a file or a directory, and rename `vendor_yaml_path` to `base_path`. Then update the corresponding docs to indicate it can be a directory or file.

Reply:
If it's a directory, process files in lexicographical order based on filename.